puppet/preprovision.sh

```diff
@@ -38,9 +38,6 @@ if [[ ! -f /etc/chassis-updated ]]; then
 	# Update apt
 	sudo apt-get update
 
-	# Install/Upgrade Puppet
-	sudo apt-get -q -y install puppet
-
```

Comment:
could have been wrapped in a conditional so that it only did this when puppet wasn't present, also if no packages are installed is the apt-get update necessary

Reply:
Yeah the conditional is a good idea.

The apt-get update was necessary when we first created this script. I can't remember the reason for it seeing it was so long ago. For now I'll revert this PR.